src/app/pages/wallets/wallet-details/pages-wallet-details.component.html

```diff
@@ -68,6 +68,12 @@
 
       <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
       <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
+
```

Comment:
Good job, looks nice. But we are also showing this when data is initially loaded. We should show loading spinner instead of the table when data is loaded. Like on wallets management page.